plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/renderer/KorteJavadocRenderer.kt

```diff
@@ -4,7 +4,7 @@
 
 package org.jetbrains.dokka.javadoc.renderer
 
-import com.soywiz.korte.*
+import korlibs.template.*
```

Comment:
The korlibs packages were simply renamed, see https://github.com/korlibs/korge/issues/1319#issuecomment-1452103500